packages/bpk-component-image/src/withLazyLoading.js

```diff
@@ -19,7 +19,7 @@
 import React from 'react';
 import { wrapDisplayName } from 'bpk-react-utils';
 
-export default function withLazyLoading(Component, document) {
+export default function withLazyLoading(Component, doc) {
```

Comment:
Why change it from document?

Reply:
Ah right yeah good question - so when myself and Shaun were debugging this we initially thought that document was being used from the global scope instead of the scope of the HOC (it wasn't). To make it very explicit we opted to rename it so as that confusion can be avoided going forward.

Reply:
When was this? I don't remember debugging it with you 😂  but I have a memory like a sieve, so..

Reply:
😂  maybe I am lying

Reply:
I feel the opposite `doc` is unclear and makes me wonder what the variable actually is. `document` is very clear and it's being shadowed so still works as expected

Reply:
Ok ill revert!